.github/workflows/python-ci.yml

```diff
@@ -15,8 +15,17 @@ name: CI using pip
 on: [push, pull_request]
 
 jobs:
+  CI_Code_Consistency:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: isort/isort-action@v1
+      - name: Suggestion to fix errors
+        if: ${{ failure() }}
+        run: |
+            echo "::notice::In project root run 'python.exe -m isort .' and commit changes to fix errors."
+            exit 1
```

Comment:
Is failure the right thing to do? Maybe stylistic issues should only be reported as warnings, like it is done with `codecov` reports?

Reply:
We can do it differently.  The reason why I decided to go for an exit 1 of the echo step is an attempt to make it more obvious that the user might want to consult the produced statement that contains a proposed fix to the issue.

The pipeline itself has already failed in the preceding step.

We should probably discuss.